docs/reference/settings/security-settings.asciidoc

```diff
@@ -2345,10 +2345,11 @@ include::ssl-settings.asciidoc[]
 [[ssl-tls-profile-settings]]
 [discrete]
 ===== Transport profile TLS/SSL settings
+NOTE: Transport profiles do not inherit TLS/SSL settings
+from the default transport. 
+
 The same settings that are available for the <<transport-tls-ssl-settings, default transport>>
-are also available for each transport profile. By default, the settings for a
-transport profile will be the same as the default transport unless they
-are specified.
+are also available for each transport profile. 
```

Comment:
The `xpack.security.transport.ssl.enabled` setting is an exception:
1. It controls whether SSL is enabled for both the default transport and any transport profiles.
2. Transport profiles do not have a corresponding setting for it.